pylint/config/callback_actions.py

```diff
@@ -238,11 +238,10 @@ def __call__(
         option_string: Optional[str] = "--generate-rcfile",
     ) -> None:
         # pylint: disable-next=fixme
-        # TODO: Optparse: Deprecate this function and raise a warning.
-        # This is (obviously) dependent on a --generate-toml-config flag.
+        # TODO: Optparse: Deprecate this after discussion about this removal has been completed.
         with warnings.catch_warnings():
             warnings.filterwarnings("ignore", category=DeprecationWarning)
-            self.run.linter.generate_config(skipsections=("COMMANDS",))
+            self.run.linter.generate_config(skipsections=("Commands",))
```

Comment:
Is the section title case insensitive ?

Reply:
No, but it now follows the actual section name as defined in `options_groups`. I'm getting the option dicts from another dictionary attribute of `PyLinter` in which we use the actual names of the groups instead of these capitalised ones.